pkg/stagemanager/stage_manager.go

```diff
@@ -567,3 +568,29 @@ func (stm *StageManager) RunAll() {
 	// stop working
 	stm.Stop()
 }
+
+// StopMosnProcess stops Mosn process via command line,
+// and it not support stop on windows.
+func StopMosnProcess(mosnConfig *v2.MOSNConfig) (err error) {
+
+	// reads mosn process pid from `mosn.pid` file.
+	var pid int
+	if pid, err = pf.GetPidFrom(mosnConfig.Pid); err != nil {
+		log.StartLogger.Errorf("[mosn stop] fail to stop MOSN, error: [%v] \n", err)
+		time.Sleep(time.Second) // waiting logs output
+		return
+	}
+
+	// sends SIGTERM to mosn process, makes it graceful exit.
+	proc, err := os.FindProcess(pid)
```

Comment:
need to handle the `err`.

Reply:
the err would be handled, as a return value, at the outside method.
```
   	if pid, err = pf.GetPidFrom(mosnConfig.Pid); err != nil {
		log.StartLogger.Errorf("[mosn stop] fail to stop MOSN, error: [%v] \n", err)
		time.Sleep(time.Second) // waiting logs output
		return
	}

```